main/src/library/LazyList.flix

```diff
@@ -594,4 +594,29 @@ namespace LazyList {
         case LList(xs)    => flatMapEHelper(f, force xs, k)
     }
 
+    ///
+    /// Returns `true` if and only if `l` contains the element `a`.
+    ///
+    /// Forces elements until `a` is found.
+    ///
+    @Time(length(l)) @Space(1)
+    pub def memberOf(a: a, l: LazyList[a]): Bool with Eq[a] = match l {
+        case ENil         => false
+        case ECons(x, xs) => if (x == a) true else memberOf(a,       xs)
+        case LCons(x, xs) => if (x == a) true else memberOf(a, force xs)
+        case LList(xs)    => memberOf(a, force xs)
+    }
+
+    ///
+    /// Returns `l2` prepended to `l1`. This operation is equivalent to `append(l2, l1)`.
+    ///
+    pub def prepend(l1: LazyList[a], l2: LazyList[a]): LazyList[a] =
```

Comment:
I am not sure why we have both prepend and append. But okay?

Reply:
I put it on the todo list at some point but I'm not sure it totally make sense either. Prepending and appending are equivalent operations with the parameters swapped - something that can be done manually by the user when needed. In short, I think we should remove it.

Reply:
Lets remove it. (and do a quick search to see if there is something like that elsewhere that should be removed too. You can use search in path in IDEA)